client/src/pages/checkout/checkout.module.css

```diff
@@ -0,0 +1,62 @@
+.details {
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction is row by default, so this can be omitted

Reply:
thank you, fixed it!